stdlib/2.7/calendar.pyi

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Iterable, Optional, Tuple
+import datetime
+
+YearType = int
+MonthType = int
+DayType = int
+WeekdayType = int
+
+LocaleType = Tuple[Optional[str], Optional[str]]
+
+_colwidth = ...  # type: int
+_spacing = ...  # type: int
```

Comment:
These are internal undocumented constants and shouldn't be defined at all. You can use ... in the signature that mention them.